crypto/src/test/java/org/hyperledger/besu/crypto/SECP256K1Test.java

```diff
@@ -280,3 +280,4 @@ public void invalidMultiLineFileThrowsInvalidIdException() throws Exception {
     KeyPairUtil.load(tempFile);
   }
 }
+‰
```

Comment:
this wasn't there before, is that hidden character or something erroneous get copied?